Update webhook reference documentation to fix incorrectly cased properties (#7137)
umbraco/UmbracoDocs: 15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md

```

* **Webhook payload property casing has changed**

> Following changes to serialization, please be aware that the properties in the payload now use camel case, rather than pascal case. For example the 'CreateDate' property becomes 'createDate'.

Reviewer: ```suggestion
 Following changes to serialization, property names in the payload now use camelCase instead of PascalCase. For example, the  'CreateDate' property is now 'createDate'.
```
